ansible_ai_connect/ai/api/model_client/tests/test_wca_client.py

```diff
@@ -385,6 +405,26 @@ def test_playbook_exp_error(self):
             model_client.explain_playbook(request, content="Some playbook")
             self.assertInLog("Caught retryable error after 1 tries.", log)
 
+    def test_playbook_exp_model_id(self):
+        request = Mock()
+        model_client = WCAClient(inference_url="http://example.com/")
+        model_client.get_api_key = Mock(return_value="some-key")
+        model_client.get_token = Mock(return_value={"access_token": "a-token"})
+        model_client.session = Mock()
+
+        self.assertion_count = 0
+
+        def get_my_model_id(user, organization_id, model_id):
+            self.assertEqual(model_id, "mymodel")
+            self.assertion_count += 1
+            return model_id
+
+        model_client.get_model_id = get_my_model_id
+
+        model_client.explain_playbook(request, content="Some playbook", model_id="mymodel")
+        self.assertGreater(self.assertion_count, 0)
```

Comment:
This is really only asserting that your mock `get_my_model_id` was called.

Would it not be better to check `WCAClient.get_model_id` was called?

Reply:
Same as above.